### PR TITLE
Adicap pipeline

### PR DESCRIPTION
**Pull request by aricohen93**
## Description

New NER pipeline to identify ADICAP codes

## Checklist

- [x] If this PR is a bug fix, the bug is documented in the test suite.
- [x] Changes were documented in the changelog (pending section).
- [x] If necessary, changes were made to the documentation (eg new pipeline).

### CODE REVIEW

File: docs/pipelines/ner/adicap.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+The compulsory characters of the ADICAP code are identified and decoded.
+These characters represent the following attributes:
+
+
+
+| Attribute [en]        | Attribute [fr]                   | Key                   |
+| --------------------- | -------------------------------- | --------------------- |
+| Sampling mode         | Mode de prelevement              | sampling_mode         |
+| Technic               | Type de technique                | technic               |
+| Organ and regions     | Appareils, organes et regions    | organ                 |
```
Comment (percevalw) [inline docs/pipelines/ner/adicap.md:20]:
```suggestion
| Organ and regions     | Appareils, organes et régions    | organ                 |
```

File: docs/pipelines/ner/adicap.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+## Usage
+
+<!-- no-check -->
+
+```python
+import spacy
+
+nlp = spacy.blank("eds")
+nlp.add_pipe("sentencizer")
```
Comment (percevalw) [inline docs/pipelines/ner/adicap.md:37]:
```suggestion
nlp.add_pipe("eds.sentences")
```

Can we use the EDS-specific pipeline ?

**Review by percevalw — DISMISSED** [review]
This is great, thanks !

We will need to work on the standardization of data preprocessing / cache in the future, but the resource is small enough at the moment so this isn't an issue.

Could you also edit the [setup.py entry points](https://github.com/aphp/edsnlp/blob/master/setup.py#L54-L86) to make the pipe visible from the outside ?

**Comment by aricohen93** [comment]
>

**Review by percevalw — APPROVED** [review]

**Comment by Thomzoy** [comment]
Looks good ! 
As the COntextualMatcher was upgraded in the same time, I took the liberty to create a new PR with your ADICAP pipeline adapted to the new Matcher, see #124

**Pull request merged**